_posts/2019-05-03-monitoring-cron-runtime-in-rails.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: post
+title: Monitoring Cron Runtime In Ruby
+categories: [ruby, whenever, redis, rake, active_admin]
```

Comment:
Lets keep categories very focussed. For example: This is not really a post on Redis.